src/renderer/shared/ui/Separator/Separator.css

```diff
@@ -2,5 +2,5 @@
   content: '';
   flex: 1;
   height: 1px;
-  background-color: theme('textColor.divider');
+  background-color: var(--divider);
```

Comment:
Turned out there is no need for `theme` and custom properties works out of the box